docs/middlewares/lscacheMiddleware.docs.md

```diff
@@ -16,7 +19,12 @@ const { lscacheMiddleware } = middlewares;
 import baseSdk from 'botify-sdk';
 
 const sdk = applyMiddleware(
-  lscacheMiddleware
+  lscacheMiddleware([
+    { // This operation will be cached by default
+      controllerId: 'Controller',
+      operationId: 'Operation',
+    }
+  ])
```

Comment:
you also need to update any docs that show an instanciation of lscacheMiddleware (introducation? readme?, ...)